mingus/containers/bar.py

```diff
@@ -49,12 +49,22 @@ def __init__(self, key="C", meter=(4, 4)):
         self.set_meter(meter)
         self.empty()
 
+    #Convenient Python operator overloading
+
+    def __iadd__(self, other):
+        self.temporal_notes.append(other)
+        return self
+
     def empty(self):
         """Empty the Bar, remove all the NoteContainers."""
+        self.temporal_notes = []
         self.bar = []
         self.current_beat = 0.0
         return self.bar
 
+    def set_chord_notes(self, chords):
+        self.chords = chords
```

Comment:
It is unclear to me why you added `chords` and `temporal_notes` to this class. It seems they are not used elsewhere.

Reply:
Not sure that I added this, maybe from a historic version of mingus? Guess it's fine to ignore